app/controllers/unattended_controller.rb

```diff
@@ -51,6 +53,7 @@ def hostgroup_template
   # i.e. /unattended/provision will render the provisioning template for the requesting host
   def host_template
     return head(:not_found) unless params[:kind].present?
+    return render_ipxe_template if ipxe_request?
```

Comment:
If there was an explicit flag support, an extra method could be only called for bootstrap options or when host parameter is present, otherwise normal workflow could follow. Not sure what ends up being cleaner code tho, you can leave this as is if it's good enough.